api/migrations/0003_fake_data.py

```diff
@@ -152,29 +190,86 @@ def _make_teacher_under_18_recipe(self):
             is_validated=self.faker.pybool,
         )
 
-    def _make_fake_coordinators(self):
-        """Makes fake coordinators."""
-        self.coordinator_recipe.make(_quantity=AMOUNT_OF_COORDINATORS)
+    def __get_random_time_or_none(self) -> time | None:
```

Comment:
double underscore? you needed name mangling here?

Reply:
Just wanted to show that it's not protected but private method. Fixed it to one underscore